google/auth/external_account.py

```diff
@@ -432,6 +448,19 @@ def _initialize_impersonated_credentials(self):
             ),
         )
 
+    def _create_default_metrics_options(self):
+        metrics_options = {}
+        if self._service_account_impersonation_url:
```

Comment:
I did them as strings instead of the bool to keep the header value consistently lowercase "true" across all languages, if I did that check, it would print "True" for the header value

Reply:
You can still simplify this using the same logic